articles/active-directory/cloud-infrastructure-entitlement-management/cloudknox-ui-remediation.md

```diff
@@ -69,31 +69,31 @@ Other options available to you:
 - **Reload**: Select this option to refresh the displayed list of roles/policies.
 - **Export CSV**: Select this option to export the displayed list of roles/policies as a comma-separated values (CSV) file. 
 
-    When the file is successfully exported, a message appears: **Exported successfully.**
+    When the file is successfully exported, a message appears: **Exported Successfully.**
 
     - Check your email for a message from the CloudKnox Customer Success Team. This email contains a link to: 
         - The **Role Policy Details** report in CSV format.
         - The **Reports** dashboard where you can configure how and when you can automatically receive reports.
-- **Create Role/Policy**: Select this option to create a new role/policy. For more information, see [Create a role/policy](cloudknox-howto-create-role-policy.md).
+- **Create Policy**: Select this option to create a new policy. For more information, see [Create a role/policy](cloudknox-howto-create-role-policy.md).
```

Comment:
Please return to "**Create Role/Policy**: Select this option to create a new role/policy." See explanation in comments.